srcpkgs/hex/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'hex'
+pkgname=hex
+version=0.1.2
+revision=1
+hostmakedepends="cargo"
+short_desc="Futuristic take on hexdump"
+maintainer="SolitudeSF <solitudesf@protonmail.com>"
+license="MIT"
+homepage="https://github.com/sitkevij/hex"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=36793a5b0c2be13974b8436952888cc0fad7651fdabd378749e9adc8518dcf1c
+nocross=yes
+
+do_build() {
+	cargo build --release
```

Comment:
please use `cargo build --release ${makejobs}`

Reply:
should all cargo based templates include this?

Reply:
imo yes.
cargo defaults to number of cpu cores. Using XBPS_MAKEJOBS, if it's set makes more sense to me.